pyproject.toml

```diff
@@ -63,9 +62,3 @@ match = "olive"
 
 [tool.semantic_release.changelog.environment]
 keep_trailing_newline = true
-
-[tool.semantic_release.remote]
-name = "origin"
-type = "github"
-ignore_token_for_push = false
-insecure = false
```

Comment:
It still creates the release in the GitHub release section right?

Reply:
Yes, those are defaults